packages/mesos/build

```diff
@@ -32,37 +33,37 @@ cp /usr/lib/x86_64-linux-gnu/libsvn_delta-1.so.1 "$libdir"
 cp /usr/lib/x86_64-linux-gnu/libsvn_subr-1.so.1 "$libdir"
 cp /usr/lib/x86_64-linux-gnu/libapr-1.so.0 "$libdir"
 cp /usr/lib/x86_64-linux-gnu/libaprutil-1.so.0 "$libdir"
+cp /usr/lib/x86_64-linux-gnu/libdb-5.3.so "$libdir"
 
 export PKG_PATH=$PKG_PATH
 systemd_master="$PKG_PATH"/dcos.target.wants_master/dcos-mesos-master.service
 mkdir -p "$(dirname "$systemd_master")"
 envsubst '$PKG_PATH' < /pkg/extra/dcos-mesos-master.service > "$systemd_master"
 
-systemd_slave="$PKG_PATH"/dcos.target.wants_slave/dcos-mesos-slave.service
-mkdir -p "$(dirname "$systemd_slave")"
-envsubst '$PKG_PATH' < /pkg/extra/dcos-mesos-slave.service > "$systemd_slave"
+systemd_agent="$PKG_PATH"/dcos.target.wants_agent/dcos-mesos-agent.service
```

Comment:
_slave and _slave_public aren't change to _agent and _agent_public currently. Way to late for that.